kafka/message.go

```diff
@@ -0,0 +1,24 @@
+package kafka
+
+import (
+	"github.com/Shopify/sarama"
+	"github.com/bsm/sarama-cluster"
+)
+
+type Message interface {
```

Comment:
Should there be godocs for the interface / struct and the functions?

Reply:
Done